test/videojs-flashls-source-handler.test.js

```diff
@@ -36,62 +36,71 @@ QUnit.module('videojs-flashls-source-handler', {
   }
 });
 
-QUnit.test('Flashls Audiotracks tests',
-  function(assert) {
-    const swfAudioTracks = [
-      {
-        id: 0,
-        enabled: true,
-        kind: '',
-        label: 'English',
-        language: ''
-      },
-      {
-        id: 1,
-        enabled: false,
-        kind: '',
-        label: 'Spanish',
-        language: ''
-      },
-      {
-        id: 2,
-        enabled: false,
-        kind: '',
-        label: 'French',
-        language: ''
-      }
-    ];
-
-    let vjsAudioTracks = [];
-    let swfAudioTrack = 0;
-
-    const tech = makeMochTech({
-      audioTracks: () => swfAudioTracks,
-      audioTrack: () => swfAudioTrack
-    }, {
-      audioTrack: (val) => swfAudioTrack = val
-    });
-
-    tech.audioTracks = () => {
-      return vjsAudioTracks;
-    };
-
-    tech.audioTracks().addTrack = (t) => {
-      vjsAudioTracks.push(t);
-    };
-
-    assert.equal(vjsAudioTracks.length, 0, 'Initial size');
-
-    setupAudioTracks(tech);
-
-    assert.equal(vjsAudioTracks.length, 3, 'Length after setupAudioTracks()');
-    assert.equal(swfAudioTrack, 0, 'Initial Audio track which is enabled');
-
-    vjsAudioTracks[1].enabled = true;
-    vjsAudioTracks[0].enabled = false;
-
-    updateAudioTrack(tech);
-
-    assert.equal(swfAudioTrack, 1, 'Enabled Audio track after switching');
-
+QUnit.test('Flashls Audiotracks tests', function(assert) {
+  const swfAudioTracks = [
+    {
+      id: 0,
+      enabled: true,
+      kind: '',
+      label: 'English',
+      language: ''
+    },
+    {
+      id: 1,
+      enabled: false,
+      kind: '',
+      label: 'Spanish',
+      language: ''
+    },
+    {
+      id: 2,
+      enabled: false,
+      kind: '',
+      label: 'French',
+      language: ''
+    }
+  ];
+
+  let vjsAudioTracks = [];
+  let swfAudioTrack = 0;
+
+  const tech = makeMochTech({
+    audioTracks: () => swfAudioTracks,
+    audioTrack: () => swfAudioTrack
+  }, {
+    audioTrack: (val) => swfAudioTrack = val
   });
+
+  tech.audioTracks = () => {
+    return vjsAudioTracks;
+  };
+
+  tech.audioTracks().addTrack = (t) => {
+    vjsAudioTracks.push(t);
+  };
+
+  assert.equal(vjsAudioTracks.length, 0, 'Initial size');
+
+  setupAudioTracks(tech);
+
+  assert.equal(vjsAudioTracks.length, 3, 'Length after setupAudioTracks()');
+  assert.equal(swfAudioTrack, 0, 'Initial Audio track which is enabled');
```

Comment:
you should be confirming here that the vjsAudioTrack with `id` of `0` has `enabled === true` and the other 2 are false instead of checking the `swfAudioTrack` because `setupAudioTracks` doesn't set anything on the swf, just sets up the vjs tracks